examples/audio_recording_merged.py

```diff
@@ -0,0 +1,115 @@
+import io
+
+import pydub  # pip install pydub
```

Comment:
Maybe add a version here. Such as 
```suggestion
import pydub  # pip install pydub==0.25.1
```

Reply:
Sure